README.md

```diff
@@ -273,8 +289,18 @@ To run the notebook you will need to start DSX Local. Below are the steps that n
         * `SSHPASSWORD`
         * `tmdb.API_KEY`
 
-      > Note: Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined two times in the notebook as they are needed in the DSX node and Livy node. Please make sure to update all the occurrence of the specific variable in the notebook.
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in the DSX node and the Livy node. 
+    
+    A Livy node refers to notebook cells that access the HDP cluster using the Livy interface. These cells can be distinquished from other notebook cells by the following tag located at the top of the cell:
 
+    Some variables such as tmdb.API_KEY, SOLR_HOST etc are defined twice in the notebook as they are needed in both the DSX node and the HDP cluster nodes. 
```

Comment:
this part seems VERY redundant.  Did you forget to delete a couple of paragraphs here?